appservice/consumers/roomserver.go

```diff
@@ -159,5 +159,33 @@ func (s *OutputRoomEventConsumer) appserviceIsInterestedInEvent(ctx context.Cont
 		}).WithError(err).Errorf("Unable to get aliases for room")
 	}
 
+	// Check if any of the members in the room match the appservice
+	// TODO: appserviceIsInterestedInEvent should be wrapped in a cache
+	// as this part can involve a lot of DB lookups
+	membershipReq := api.QueryMembershipsForRoomRequest{
```

Comment:
This seems to be the wrong function to call. It expects a userId (presumably because this call is expected to handle `/members`). I'd like to just get membership of the room without the context of a sender user.

Reply:
We're using `QueryStateAfterEvents` which seems to do the right thing.